<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSelectConnection</class>
 <widget class="QDialog" name="frmSelectConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>260</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>登录界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">frmSelectConnection
{
	
	background-color: rgb(144, 148, 248);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>选择语言</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="rbChinese">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>简体中文</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbEnglish">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>English</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbTwLan">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>繁體中文</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSwitch">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>切换</string>
        </property>
        <property name="icon">
         <iconset resource="../recFile/myRes.qrc">
          <normaloff>:/Icons/Auto.png</normaloff>:/Icons/Auto.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>选择测试模式</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnMeter">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>仪表模式(&amp;M)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAnalysis">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>分析模式(&amp;S)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRansonace">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>谐振模式（&amp;R）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMulitChannel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>多通道测试(&amp;C)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAppMode">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>App 模式（&amp;P）</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpConnection">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>选择连接方式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbGpib">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>如果选择GPIB连接仪器，请设置地址范围1~32</string>
          </property>
          <property name="text">
           <string>&amp;GPIB</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbLan">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>如果连接6500请输入IP地址形如：192.168.1.102,6500
如果连接4300请输入Ip地址形如：192.168.1.102,4300</string>
          </property>
          <property name="text">
           <string>&amp;LAN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbUsb">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&amp;USB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblAddress">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>地址(&amp;A)：</string>
          </property>
          <property name="buddy">
           <cstring>txtAddress</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtAddress">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;如果选择GPIB连接仪器，请设置地址范围1~32&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;如果连接LAN连接6500请输入IP地址形如：192.168.1.102&lt;/p&gt;&lt;p&gt;如果连接LAN连接4300请输入Ip地址形如：192.168.1.102&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbModel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>4300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6500</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnTest">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>连接(&amp;T)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInformation">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>如果选择GPIB连接仪器，请设置地址范围1~32

如果连接LAN连接6500请输入IP地址形如：192.168.1.102,6500
如果连接LAN连接4300请输入Ip地址形如：192.168.1.102,4300</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../recFile/myRes.qrc"/>
 </resources>
 <connections/>
</ui>
